src/styles/blocks/stars.css

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  margin: 0;
```

Comment:
why do you need to reset them? is there default values? also, it breaks BEM rule